nanome/_internal/_ui/_button.py

```diff
@@ -1,5 +1,5 @@
 from . import _UIBase
-from nanome.util import Vector3, Color
+from nanome.util import Vector3, Color, Logs
```

Comment:
I don't see `Logs` being used in this file or image, did you mean to change the `nanome.util.Logs` in the `except` for registering a hook?